src/main/java/gregtech/common/covers/CoverConveyor.java

```diff
@@ -350,7 +350,7 @@ public GroupItemInfo(Object filterSlot, Set<ItemStackKey> itemStackTypes, int to
 
     @Override
     public boolean canAttach() {
-        return coverHolder.getCapability(CapabilityItemHandler.ITEM_HANDLER_CAPABILITY, attachedSide) != null;
+        return coverHolder.getCapability(CapabilityItemHandler.ITEM_HANDLER_CAPABILITY, null) != null;
```

Comment:
Why is this required? This does not seems right at all.

Reply:
`canAttach` is called before placing, so if `allowInputFromOutputSide` is still set on the output side, machines that only have fluid or items output slots, will expose only one of the capabilities even though the machine may accept both.
If we check the side capability, you wont be able to place a conveyor on the output side of fluid extractor.
Now that i explained it, i think its not really needed.